search_service/proxy/elasticsearch.py

```diff
@@ -98,7 +100,10 @@ def _get_search_result(self, page_index: int,
                 result = {}
                 for attr, val in es_payload.items():
                     if attr in model.get_attrs():
-                        result[attr] = val
+                        if attr == "badges":
```

Comment:
This should be made more generic if this is the way we want to handle it (ie check if it's a class instead?)

Reply:
But I did this for now since I wasn't sure what people actually thought about this